bin/ytdl.js

```diff
@@ -283,6 +283,7 @@ if (opts.infoJson) {
   } else {
     var readStream = ytdl(opts.url, ytdlOptions);
     var liveBroadcast = false;
+    var is_stdout_mutable = (process.stdout && process.stdout.cursorTo && process.stdout.clearLine);
```

Comment:
Should be named in camelCase style to preserve consistency with the codebase

No need to wrap statement with parenthesis